content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,157 @@
+
+### Configuring Plugins in Falco
+
+Plugins are configured in yaml via new `falco.yaml` properties `plugins` and `load_plugins`. Here's an example:
+
+```yaml
+plugins:
+  - name: cloudtrail
+    library_path: cloudtrail/plugin.so
+    init_config: "..."
+    open_params: "..."
+  - name: json
+    library_path: json/plugin.so
+    init_config: ""
+    open_params: ""
+
+# Optional
+load_plugins: [cloudtrail, http_json]
+```    
+
+The `plugins` property in falco.yaml defines the set of plugins that can be loaded by Falco. It contains a list of plugin definitions. Each plugin definition has:
+
+* `name`: the name of the plugin
+* `library_path`: a path to the plugin shared library
+* `init_config`: A configuration string (typically JSON) that is passed to the plugin when the plugin is initialized.
+* `open_params`: A configuration string that is passed to the plugin when the plugin `open()s` a new stream of events.
+
+By default, every plugin in the plugins list will be loaded. If present, the `load_plugins` property contains specific name entries from `plugins` that are loaded instead.
+
+#### Running Plugins in Falco
+
+For now, when plugin support is enabled, Falco's core support for syscalls is disabled. Additionally, only a single source plugin can be loaded at once. If multiple source plugins are loaded, falco will return an error. Any number of extractor plugins can be loaded at once.
+
+We do plan on relaxing these restrictions to allow running multiple source plugins at once, once we tackle the concurrency, fairness, and resource utilization issues that result from having multiple streams of events being processed by a single falco rules engine.
+
+### Cloudtrail Plugin
+
+Along with the introduction of plugins support, we've written a source plugin [cloudtrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail) that can read cloudtrail logs and pass them to the rules engine as events.
+
+The plugin can be configured to obtain log files in a number of ways, depending on the `open_params` value in `falco.yaml`:
+
+* A [S3](https://aws.amazon.com/s3/) bucket: `s3://<S3 Bucket Name>[/<Optional Prefix>]`.
+* A [SQS](https://aws.amazon.com/sqs/) queue that passes along [SNS notifications](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/configure-sns-notifications-for-cloudtrail.html) about new log files: `sqs://<SQS Queue Name>`
+* A local filesystem path: `Any path`.
+
+The plugin also exports fields that extract information from a cloudtrail event, such as the event time, the aws region, S3 bucket/EC2 instance name, etc.
+
+The plugin uses the same authentication mechanisms used by the [AWS Go SDK](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials):
+
+* **Environment Variables**: specify the aws region with `AWS_REGION=xxx`, the access key id with `AWS_ACCESS_KEY_ID=xxx`, and the secret key with `AWS_SECRET_ACCESS_KEY=xxx`.
```

Comment:
Specify the AWS region with `AWS_REGION=xxx`, the access key id with `AWS_ACCESS_KEY_ID=xxx`, and the secret key with `AWS_SECRET_ACCESS_KEY=xxx`.